src/Components/App/AppS.scss

```diff
@@ -0,0 +1,4 @@
+.App {
+  height: 100vh;
+  width: 100vw;
```

Comment:
be sure to account for your header px size, also do we want this to be a full page or for mobile proportions?